libs/ui-lib/lib/ocm/components/clusterWizard/HostDiscovery.tsx

```diff
@@ -65,25 +63,11 @@ const HostDiscovery = ({ cluster }: { cluster: Cluster }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
     [], // just once, Formik does not reinitialize
   );
-  const { supportedPlatformIntegration } = useClusterSupportedPlatforms(cluster.id);
 
   const onSubmit: FormikConfig<HostDiscoveryValues>['onSubmit'] = async (values) => {
     clearAlerts();
 
-    const platformToIntegrate = values.usePlatformIntegration
-      ? supportedPlatformIntegration === 'no-active-integrations'
-        ? undefined
-        : supportedPlatformIntegration
-      : undefined;
-
     const params: V2ClusterUpdateParams = {};
-    if (cluster.platform?.type !== 'oci') {
-      HostDiscoveryService.setPlatform(
```

Comment:
`HostDiscoveryService.setPlatform` should be removed, it's not used anymore.

Note how it was setting `none` or `baremetal` depending on the value for `userManagedNetworking`, you may need to do the same in the new code from ClusterDetailsForm.